torch/masked/_ops.py

```diff
@@ -1579,7 +1579,11 @@ def norm(
     if mask_input.layout == torch.strided:
         dim_ = _canonical_dim(dim, input.ndim)
         return torch.linalg.vector_norm(
-            mask_input, ord, dim_, bool(keepdim), dtype=dtype
+            mask_input,
+            ord if ord is not None else 2.0,
```

Comment:
This is a typing error of the definition (likely from gen_pyi.py) It clearly support None as input since it didn't error before.

Reply:
reverted. I'm looking into a change in gen_pyi.py